modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java

```diff
@@ -1060,6 +1210,20 @@ public AssetEntryModel newAssetEntryModel(MBThreadModel mbThreadModel) {
 			String.valueOf(mbThreadModel.getRootMessageId()));
 	}
 
+	public AssetEntryModel newAssetEntryModel(
+		ObjectEntryModel objectEntryModel) {
+
+		return newAssetEntryModel(
+			objectEntryModel.getGroupId(), objectEntryModel.getCreateDate(),
+			objectEntryModel.getModifiedDate(),
+			getClassNameId(
+				ObjectDefinition.class.getName() + "#" +
```

Comment:
Can you change this code to use `objectEntryModel.getModelClassName()`, we centralize this logic in this method. More context on this [PR](https://github.com/liferay-bpm/liferay-portal/pull/3782).

Reply:
We can't use this method, because we do not have access to local service on DataFactory. So it would throw a `NullPointerException`. But I changed it to use the new prefix constant.